.changeset/moody-bobcats-travel.md

```diff
@@ -0,0 +1,28 @@
+---
+'@seek/logger': major
```

Comment:
Should this be a `minor` version given previous "feature" releases are minor version changes,  
_**OR**_ should this be a `major` version since the "feature" affects what's being logged (certain headers are omitted by default)?

```suggestion
'@seek/logger': minor
```